v2/spiffejwt/svid.go

```diff
@@ -0,0 +1,36 @@
+package spiffejwt
+
+import (
+	"crypto"
+	"time"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// KeyStore is an interface used to locate the public key for a signed JWT-SVID.
+type KeyStore interface {
```

Comment:
Should this feature be something that spiffebundle.Bundle provides? Do we really want a `Mint` function in this package? Curious to hear more about the intended use of the functions in this particular file

Reply:
`Bundle` does provide `KeyStore`. Other things could also provide `KeyStore`. I didn't want to be proscriptive about how the keys were provided to the validator...